src/test/mediapipe/python/scripts/symmetric_increment.py

```diff
@@ -0,0 +1,36 @@
+#*****************************************************************************
```

Comment:
Suggestion: separate python test scripts for negative and positive?

Reply:
I added prefixes to scripts that fail somewhere in the execution. Do you mean something more?

Reply:
I think that's enough